### ISSUE TITLE
Game crashes when opening GUI

### ISSUE THREAD

**Issue by Ollix27** [TeamPneumatic/pnc-repressurized#891]
*For feature requests, just erase this template and clearly describe the feature you'd like to see*

1.16.5 Minecraft Version

36.2.0 Forge Version

2. 14 Mod Version 

My game is crahing when i try to open the GUI Errorcode: Tag forge:tools/wrench used before it was bound

No other mods.

**Comment by ghost** [comment]
Reproduced opening Thermo Plant or Charging Station GUI. As with the original report, this was with no other mods. Mod version pneumaticcraft-repressurized-1.16.5-2.14.2-246.

Full crash report: https://gist.github.com/mc-ganduron/c734022bc571e08410c3ab51ea9db120

Excerpt:

```
java.lang.IllegalStateException: Tag forge:tools/wrench used before it was bound

	at net.minecraft.tags.TagRegistry$NamedTag.func_232944_c_(TagRegistry.java:131) ~[?:?] {re:classloading,pl:accesstransformer:B}

	at net.minecraft.tags.TagRegistry$NamedTag.func_230235_a_(TagRegistry.java:142) ~[?:?] {re:classloading,pl:accesstransformer:B}

	at net.minecraft.item.Item.func_206844_a(Item.java:343) ~[?:?] {re:classloading,xf:fml:forge:crossbows.8}

	at me.desht.pneumaticcraft.common.thirdparty.ModdedWrenchUtils.isModdedWrench(ModdedWrenchUtils.java:47) ~[?:1.16.5-2.14.2-246] {re:classloading}

	at me.desht.pneumaticcraft.common.event.EventHandlerPneumaticCraft.onModdedWrenchBlock(EventHandlerPneumaticCraft.java:174) ~[?:1.16.5-2.14.2-246] {re:classloading}

	at net.minecraftforge.eventbus.ASMEventHandler_22_EventHandlerPneumaticCraft_onModdedWrenchBlock_RightClickBlock.invoke(.dynamic) ~[?:?] {}

	at net.minecraftforge.eventbus.ASMEventHandler.invoke(ASMEventHandler.java:85) ~[eventbus-4.0.0.jar:?] {}

```

**Comment by desht** [comment]
Oops, that's what I get for not testing things properly... Will get this fixed in a release tomorrow.

**Comment by Ollix27** [comment]
Thanks, It Works now.

**Comment by desht** [comment]
Fixed in 2.14.2 release
